lib/origami/parser.rb

```diff
@@ -57,7 +57,7 @@ def initialize(options = {}) #:nodoc:
             #Default options values
             @options =
             {
-                verbosity: VERBOSE_INFO, # Verbose level.
+                verbosity: VERBOSE_QUIET, # Verbose level.
```

Comment:
This is just changing the default option for verbosity within the gem, we should avoid changing a default without good reason. 
As it's an option though, looks like we can change the log level where we use the gem rather than here